chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -1407,25 +1407,32 @@ we have seen before can be expressed in this compressed tensor notation:
 * $\mathrm{tr}(\mathbf{A}) = \sum_i a_{ii}$
 -->
 
-*dịch đoạn phía trên*
+* $\mathbf{v} \cdot \mathbf{w} = \sum_i v_iw_i$
+* $\|\mathbf{v}\|_2^{2} = \sum_i v_iv_i$
+* $(\mathbf{A}\mathbf{v})_i = \sum_j a_{ij}v_j$
+* $(\mathbf{A}\mathbf{B})_{ik} = \sum_j a_{ij}b_{jk}$
+* $\mathrm{tr}(\mathbf{A}) = \sum_i a_{ii}$
```

Comment:
Lần sau thấy đonaj toàn toán thế này em bỏ hai dòng open và close block comment là được. Đỡ mất công check lại xem có copy paste sai đâu không.